GDAPI/GDAPI/Enumerations/GeometryDash/ObjectProperty.cs

```diff
@@ -68,10 +68,10 @@ public enum ObjectProperty
         EL1 = 20,
         /// <summary>Represents the Main Color Channel value of the <see cref="LevelObject"/>.</summary>
         [ObjectPropertyIntType]
-        Color1 = 21,
+        Color1ID = 21,
```

Comment:
```suggestion
        ColorID1 = 21,
```

Reply:
No because it's the Color 1's ID, similarly below.